actions/src/cla_check/claCheck.js

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview File to check if PR Author has signed the CLA
+ */
+
+const core = require('@actions/core');
+const { context, GitHub } = require('@actions/github');
+const { google } = require('googleapis');
+const RANGE = 'Usernames';
```

Comment:
I would use better name here, range is confusing.

Reply:
TODO: Update the range to range name used in CLA sheet

Reply:
Is this name better
`SPREADSHEET_RANGE`

Reply:
Still not sure to what range is referring, the columns and lines?

Reply:
The range refers to a column in a spreadsheet.

Example: Column A is being used in the program right now.
![image](https://user-images.githubusercontent.com/30050862/113718424-3d5e7600-970a-11eb-9c90-02f18c1ece40.png)

Reply:
`SPREADSHEET_COLUMN` then?

Reply:
I removed that variable since we're using it in one place only